packages/celotool/genesis_rc0.json

```diff
@@ -0,0 +1,87 @@
+  "difficulty": "0x1",
+  "coinbase": "0x0000000000000000000000000000000000000000",
+  "alloc": {
```

Comment:
I think we normally give validators some gold here otherwise the migrations might fail

Reply:
We won't be running the migration that registers validators in order to better mimic a mainnet deployment.